charts/sn-platform/values.yaml

```diff
@@ -2124,12 +2124,15 @@ vault:
       statsd_address: ""
     bankVaults:
       probe:
-        readinessProbe:
-          failureThreshold: 2
-          httpGet:
-            path: "/v1/sys/init"
-            port: "api-port"
-            scheme: "HTTP"
+        readinessProbe: {}
+        #  Note: To use readinessProbe, you can uncomment below configs
+        #  If you want to use readinessProbe with Istio enabled, you have to chagne port from "api-port" to "http-api-port"
```

Comment:
typo: `chagne` -> `change`

Reply:
fixed in https://github.com/streamnative/charts/pull/647/commits/4ac1f5eedefa8de42e323e2bc0520906425ab509